.github/workflows/main.yml

```
@@ -92,6 +92,24 @@ jobs:
     - run: cargo test ${{ matrix.no_run }} --manifest-path cc-test/Cargo.toml --target ${{ matrix.target }} --features parallel
     - run: cargo test ${{ matrix.no_run }} --manifest-path cc-test/Cargo.toml --target ${{ matrix.target }} --release
 
+  test-tvos:
```

I think "check" is a better name than "test" since no test is actually run:

```suggestion
  check-tvos:
```